soapboxpb/application.proto

```diff
@@ -29,17 +29,22 @@ enum CreationState {
 
 message Application {
     int32 id = 1;
-    string name = 2;
-    string description = 3;
-    string external_dns = 4;
-    string github_repo_url = 5;
-    string dockerfile_path = 6;
-    string entrypoint_override = 7;
-    ApplicationType type = 8;
-    google.protobuf.Timestamp created_at = 9;
-    string slug = 10;
-    string internal_dns = 11;
-    CreationState creation_state = 12;
+    int32 user_id = 2;
```

Comment:
Just for your edification, once protobufs are out in the wild existing field definitions should be considered immutable, because otherwise you could break clients and servers communication, but you can always append new fields. Protobufs guarantees that new versions of messages are backward compatible so long as the old fields stay the same. So here you would have done `int32 user_id = 13`. But since we're pre-release and still rapidly evolving it's ok to leave it as-is.

Reply:
Okay, good to know!